Database/Updates/World/010_2017_04_24_destination_portal.sql

```diff
@@ -0,0 +1,15 @@
+INSERT INTO `portal_destination` (`weenieClassId`, `cell`, `x`, `y`, `z`, `qx`, `qy`, `qz`, `qw`, `min_lvl`, `max_lvl`) VALUES
+(1124, 32965104, 40, -140, 0.005, 1, 0, 0, 0, 0, 0),
+(1096, 3662938154, 135.1000061, 42.70000076, 34, -0.96592581, 0, 0, -0.25881913, 0, 0),
```

Comment:
I think you have your qx and qz values backwards, here.  QX rotation is like doing barrel-roll rotations.  No bueno.